claimManagement/src/chfDev/java/org/openimis/imisclaims/Escape.java

```diff
@@ -0,0 +1,11 @@
+package org.openimis.imisclaims;
+
+public class Escape {
+    public boolean CheckCHFID(String InsureeNumber) {
```

Comment:
CHF is using modulo 7 validation. Please add same validation as in Policies: https://github.com/openimis/policies_android_app_java/blob/develop/app/src/chf/java/org/openimis/imispolicies/Escape.java

Reply:
I added the CHFID validation from Policies app to chfDev build